app/models/student_identifier_type.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+class StudentIdentifierType < ActiveRecord::Base
+  default_scope { where(deleted_at: nil) }
+
+  belongs_to :organization, -> { unscope(where: :deleted_at) }
+
+  enum content_type: [:text, :number, :email]
```

Comment:
This is minor, but that "Please don't ask..." notice isn't in a great spot below the save button. That seems like an important thing people should see, so maybe have that at the top of the form?